Examples/Framework/include/ActsExamples/Framework/WhiteBoard.hpp

```diff
@@ -38,28 +37,20 @@ class WhiteBoard {
                  Acts::getDefaultLogger("WhiteBoard", Acts::Logging::INFO),
              std::unordered_map<std::string, std::string> objectAliases = {});
 
-  // A WhiteBoard holds unique elements and can not be copied
   WhiteBoard(const WhiteBoard& other) = delete;
   WhiteBoard& operator=(const WhiteBoard&) = delete;
 
+  WhiteBoard(WhiteBoard&& other) = default;
+  WhiteBoard& operator=(WhiteBoard&& other) = default;
```

Comment:
I think if we do not use `delete` and `default` we get the same behavior by default?

Reply:
Because of the `std::unique_ptr<Logger>` as member? I think its not bad to have it explicit...